x/shape/fromfile.go

```diff
@@ -562,7 +564,14 @@ func (f *InferredInfo) Visit(n ast.Node) ast.Visitor {
 		for _, imp := range t.Imports {
 			pkgImportName := strings.Trim(imp.Path.Value, "\"")
 			if imp.Name != nil {
-				f.packageNameToPackageImport[imp.Name.String()] = pkgImportName
+				if imp.Name.String() == "." {
+					// Dot import - types from this package appear without package prefix
+					// We need to track this for proper type attribution
+					f.packageNameToPackageImport["."] = pkgImportName
```

Comment:
golang allows to have multiple dot imports ie
```
import (
	"fmt"
	. "github.com/widmogrod/mkunion/f"
	. "github.com/widmogrod/mkunion/x/shared"
)
```

Please demonstrate that it can work in such cases also.